onnx/defs/rnn/defs.cc

```diff
@@ -161,22 +193,36 @@ custom implementation such as CuDNN.
 `WBb[iofc]` - W bias vectors for backward input, output, forget, and cell gates
 `RBb[iofc]` - R bias vectors for backward input, output, forget, and cell gates
 `PB[iof]`  - P peephole weight vector for backward input, output, and forget gates
-`tanh(X)` - hyperbolic tangent of X
-`sigmoid(X)` - 1 / (1 + e^-X)
 `H` - Hidden state
 `num_directions` - 2 if direction == bidirectional else 1
 
+Activation functions:
+  ReLU(x)                - max(0, x)
+  tanh(x)                - (1 - e^{-2x})/(1 + e^{-2x})
+  sigmoid(x)             - 1/(1 + e^{-x})
+  (NOTE: Below are optional)
+  linear(x)              - alpha*x + beta
+  leakyReLU(x)           - x if x >= 0 else alpha * x
```

Comment:
so, how does user specify alpha/beta?

Reply:
Using the two optional attributes alpha and beta.